src/main/java/seedu/address/model/status/Status.java

```diff
@@ -9,8 +9,8 @@
  */
 public class Status {
 
-    public static final String MESSAGE_CONSTRAINTS = "The statuses are only none, non-urgent and urgent. Please use "
-            + "only one of them";
+    public static final String MESSAGE_CONSTRAINTS = "The Statuses are None, Non_Urgent and Urgent. Please use "
```

Comment:
I am not too sure about putting none here and capitalising each word, as it may cause the user to unnecessarily capitalise the status, which can reduce speed.